site/gdocs/topic-page.scss

```diff
@@ -173,12 +173,22 @@
     }
 }
 
+// small amount of extra padding to match figma designs which is 40px total for key-insights heading
+.article-block__heading + .wp-block-owid-key-insights {
+    margin-top: 8px;
+}
+
 .article-block__gray-section {
     > h1 {
-        text-align: center;
         // Countering the gray-section padding to make this have 32px 16px
         margin-top: -16px;
         margin-bottom: 16px;
+
+        // Special exception for the "Explore data on Blah" heading (which is a gdoc .article-block__heading)
+        // so that it's left-aligned like the rest of the topic page component headings
+        &[id^="explore"] {
+            grid-column-start: 2;
+        }
```

Comment:
I think we should make this apply to topic pages only (e.g. `.centered-article-container--topic-page`) so that the override doesn't leak onto articles (granted, it is somewhat unlikely that we'll have a h1 in a gray section starting with "Explore..." in an article...)